src/applications/check-in/day-of/pages/ArrivedAtFacility.jsx

```diff
@@ -16,40 +19,91 @@ const ArrivedAtFacility = props => {
   const { goToNextPage, getCurrentPageFromRouter } = useFormRouting(router);
   const currentPage = getCurrentPageFromRouter();
 
+  const pageType = 'arrived-at-facility';
+
+  const testID = 'arrived-page';
+
   const bodyText = (
     <>
       <p>{t('we-ask-this-question-because-staff-can-call-you')}</p>
     </>
   );
 
-  const yesAction = () => {
+  // const bodyText = t('we-ask-this-question-because-staff-can-call-you');
+
+  const selectCurrentContext = useMemo(makeSelectCurrentContext, []);
+  const { setECheckinStartedCalled } = useSelector(selectCurrentContext);
+  const selectApp = useMemo(makeSelectApp, []);
+  const { app } = useSelector(selectApp);
```

Comment:
Not a huge deal and nothing you need to change for this PR, but for pages that are app specific like this, you don't need to select the app. You could just pass `APP_NAMES.CHECK_IN` to the slug and simplify the logic a bit.
Something like:
```
recordEvent({
      event: createAnalyticsSlug(
        `${answer}-to-${pageType}${
          setECheckinStartedCalled ? '' : '-45MR'
        }-clicked`,
        'nav',
        APP_NAMES.CHECK_IN,
      ),
    });
```